python/ray/experimental/dag/class_node.py

```diff
@@ -133,6 +133,15 @@ def bind(self, *args, **kwargs):
         self._actor._last_call = node
         return node
 
+    def remote(self, *args, **kwargs):
+        raise AttributeError(
+            "All DAGNode types act as intermediate representation (IR) of a "
```

Comment:
Isn't it more confusing to have this defined? It can show up in IDE, etc. and confuse people if they don't read carefully.

Reply:
How about we define this in `__getattr__`() instead, so it doesn't show up in autocomplete?

Reply:
i see your point, sure let me move it to `__getattr__`

Reply:
`__getattr__` SGTM